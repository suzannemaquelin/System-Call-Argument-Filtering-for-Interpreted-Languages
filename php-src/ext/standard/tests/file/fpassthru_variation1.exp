*** Testing fpassthru() : usage variations ***

Arg value 0 

Warning: fpassthru() expects parameter 1 to be resource, integer given in %s on line %d
bool(false)

Arg value 1 

Warning: fpassthru() expects parameter 1 to be resource, integer given in %s on line %d
bool(false)

Arg value 12345 

Warning: fpassthru() expects parameter 1 to be resource, integer given in %s on line %d
bool(false)

Arg value -2345 

Warning: fpassthru() expects parameter 1 to be resource, integer given in %s on line %d
bool(false)

Arg value 10.5 

Warning: fpassthru() expects parameter 1 to be resource, float given in %s on line %d
bool(false)

Arg value -10.5 

Warning: fpassthru() expects parameter 1 to be resource, float given in %s on line %d
bool(false)

Arg value 101234567000 

Warning: fpassthru() expects parameter 1 to be resource, float given in %s on line %d
bool(false)

Arg value 1.07654321E-9 

Warning: fpassthru() expects parameter 1 to be resource, float given in %s on line %d
bool(false)

Arg value 0.5 

Warning: fpassthru() expects parameter 1 to be resource, float given in %s on line %d
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in %s on line %d
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in %s on line %d
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in %s on line %d
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in %s on line %d
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in %s on line %d
bool(false)

Arg value 1 

Warning: fpassthru() expects parameter 1 to be resource, boolean given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, boolean given in %s on line %d
bool(false)

Arg value 1 

Warning: fpassthru() expects parameter 1 to be resource, boolean given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, boolean given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, string given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, string given in %s on line %d
bool(false)

Arg value string 

Warning: fpassthru() expects parameter 1 to be resource, string given in %s on line %d
bool(false)

Arg value string 

Warning: fpassthru() expects parameter 1 to be resource, string given in %s on line %d
bool(false)

Arg value testClass 

Warning: fpassthru() expects parameter 1 to be resource, object given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in %s on line %d
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in %s on line %d
bool(false)
Done