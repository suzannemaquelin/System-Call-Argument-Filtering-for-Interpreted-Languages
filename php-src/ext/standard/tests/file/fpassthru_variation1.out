Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
*** Testing fpassthru() : usage variations ***

Arg value 0 

Warning: fpassthru() expects parameter 1 to be resource, integer given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 1 

Warning: fpassthru() expects parameter 1 to be resource, integer given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 12345 

Warning: fpassthru() expects parameter 1 to be resource, integer given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value -2345 

Warning: fpassthru() expects parameter 1 to be resource, integer given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 10.5 

Warning: fpassthru() expects parameter 1 to be resource, float given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value -10.5 

Warning: fpassthru() expects parameter 1 to be resource, float given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 101234567000 

Warning: fpassthru() expects parameter 1 to be resource, float given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 1.07654321E-9 

Warning: fpassthru() expects parameter 1 to be resource, float given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 0.5 

Warning: fpassthru() expects parameter 1 to be resource, float given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value Array 

Warning: fpassthru() expects parameter 1 to be resource, array given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 1 

Warning: fpassthru() expects parameter 1 to be resource, boolean given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, boolean given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value 1 

Warning: fpassthru() expects parameter 1 to be resource, boolean given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, boolean given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, string given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, string given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value string 

Warning: fpassthru() expects parameter 1 to be resource, string given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value string 

Warning: fpassthru() expects parameter 1 to be resource, string given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value testClass 

Warning: fpassthru() expects parameter 1 to be resource, object given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)

Arg value  

Warning: fpassthru() expects parameter 1 to be resource, null given in /root/php-src/ext/standard/tests/file/fpassthru_variation1.php on line 76
bool(false)
Done